ibm/service/contextbasedrestrictions/resource_ibm_cbr_zone.go

```diff
@@ -110,36 +115,6 @@ func ResourceIBMCbrZone() *schema.Resource {
 							Optional:    true,
 							Description: "The IP address.",
 						},
-						"ref": &schema.Schema{
```

Comment:
why this "ref" been removed from schema?

Reply:
CBR API only allowed to include "ref" in [addresses](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3971/files#diff-6b5bf268286a585272f1b259bf479cb72deffd67282020b1a8e1bc83b7df1edaL48) object, other than [excluded](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3971/files#diff-6b5bf268286a585272f1b259bf479cb72deffd67282020b1a8e1bc83b7df1edaL97) object.